packages/x-components/src/x-modules/history-queries/store/actions/load-history-queries-from-browser-storage.action.ts

```diff
@@ -13,6 +13,7 @@ import { HistoryQueriesXStoreModule } from '../types';
 // eslint-disable-next-line max-len
 export const loadHistoryQueriesFromBrowserStorage: HistoryQueriesXStoreModule['actions']['loadHistoryQueriesFromBrowserStorage'] =
   ({ commit, getters }) => {
-    const historyQueries = localStorageService.getItem<HistoryQuery[]>(getters.storageKey) ?? [];
-    commit('setHistoryQueries', historyQueries);
+    const historyQueries =
+      localStorageService.getItem<HistoryQuery[] | HistoryQuery>(getters.storageKey) ?? [];
+    commit('setHistoryQueries', [historyQueries].flat());
```

Comment:
I don't understand the need for this change. AFAIK we always store the history queries as an [array](https://github.com/empathyco/x/blob/main/packages/x-components/src/x-modules/history-queries/store/actions/add-query-to-history.action.ts#L34+L42) So I don't think we need the flatting.